components/ShowNutritionalTable.tsx

```diff
@@ -13,10 +13,24 @@ export default function ShowNutritionalTable() {
   const nutriments = useSelector(
     (state: RootState) => state.recipe.recipes["empty_recipe"].nutriments
   );
-
+  const modifiedNutriments = useSelector(
+    (state: RootState) => state.recipe.recipes["modifiedRecipe"].nutriments
+  );
+  const modifiedNutriscore = useSelector(
+    (state: RootState) => state.recipe.recipes["modifiedRecipe"].nutriscore
+  );
+ 
   const [open, setOpen] = React.useState(false);
 
-  if (!nutriments || !nutriscore) {
+  console.log(modifiedNutriments, "modifiedNutriments")
+  console.log(nutriments, "nutriments")
+
+  if (
+    !nutriments ||
+    !nutriscore ||
+    !Object.keys(modifiedNutriments).length ||
+    !modifiedNutriscore
+  ) {
```

Comment:
That's a bit weird to do not display anything just because either the `nutriscore` or the `modifiedNutriscore` is missing